docs/load-test.md

```diff
@@ -0,0 +1,29 @@
+## Test simulations
+
+The platform was tested using following test simulations:
```

Comment:
No need for describing the purpose of stress tests. Instead, use this "intro" to describe the testing environment. Since it is missing at the moment, put the statement: `Testing environment to be determined.`.